### PR TITLE
Aetherwhisp engineering revamp plus a lot of bugfixes

### LINKED ISSUE #1283: Aetherwhisp - Toxins Combustion chamber airlock names are backwards in comparison to access console

### LINKED ISSUE #1350: Aetherwhisp Munitions bay has cargo protolathe
As the name states, it was never replaced with a muni protolathe.

### PR DESCRIPTION
**Pull request by IndusRobot**
## About The Pull Request

Aetherwhisp has changed significantly since its original release! (Over 50%) This marks a new version of the Aetherwhisp which is tracked internally as 8.0

Wow! New screenshots!

Deck 2 top deck 

aetherfullengirevamp2

Deck 1 bottom deck

aetherfullengirevamp1

## Why It's Good For The Game

Fixes the following bugs, some small requests 

- Reports that engineering is still cramped and rooms are generally thrown around. Time for a proper internal department reshuffle!
- Add a station alert console to the bridge 
- Move food processor out of freezer
- Add wire knot to Bridge maint APC
- Cargo lift request button inaccessible by a wall
- Cargo checkpoint APC no power 
- AI upload turret control tile has rad immunity on the bridge. Remove rad immunity from AI upload and vault
- Add cargo button for shuttle shutter id = "ss_hh_cargodoor_top"
- Add specimen chamber to xenobiology 
- Missing camera coverage in xeno bio
- Double APC in bridge maint
- Armory missing firelocks
- Bring back armor plating
- Cargo does not have enough shuttle displays, one in QM
- Use a wrap sorting object to mark the beginning of disposals, and fix junk sorting
- Add new engi warehouse for some singulo equipment moved inside
- Add two grounding rods to engi warehouse
- Potentially add more space to the stormdrive to allow dev work 
- Reverse sm burn input output 
- Add TEG arrows 
- Remove any maintenance abandoned crates and lootcrate spawners, and readd 2 change spawners in sealed compartments
- Remove the last munitech job slot, keep MAA
- Add engineering access to the first two doors in munitions, due to lasers being zero-maintenance and a button switch. Reminder to remove this engineering access if laser UI and functionality are upgraded to require munitechs or if the ship switches to NT
- Separate munihangar storage into two rooms
- Improved overmap thruster speed
- Coerce the science elevator specifically to let lizards go up by area name shenanigans that don't use s
- Replace AI telescreen with viewscreen
- Improve foot traffic in medbay by adding double doors to storage

Fixes #1283 
Fixes #1350 

## Changelog
:cl:
add: Aetherwhisp, new engineering department layout
del: Removed the last munitions technician job slot due to lack of munitions content. Reminder that new job slots can be added at the identification console
tweak: Improved Aetherwhisp thruster maneuverability 
/:cl:

### CODE REVIEW

**Comment by jzburda** [comment]
Testmerge added a whole bunch of rooms around the ship, moved Toxins catapult, and most importantly, removed toxins storage removing the 3 oxygen canisters to speedfully complete a sciencecap.

**Comment by moonmage04** [comment]
(Sorry in advance if this is the wrong place to write this I don't know where)
What I assume is supposed to be mixers in the atmos mix loop is filters.
Can you maybe move the TEG one tile down?

**Comment by SerynEngi** [comment]
> Testmerge added a whole bunch of (unfinished) rooms around the ship, moved Toxins catapult, and removed toxins storage, which removed the 3 oxygen canisters required to (speedily) complete a sciencecap. 

The toxins catapult was moved to the lower deck, at my request, due to noting that detonations on upper Z-levels that are sufficient to destroy the testing area tend to drop subsequent bombs into lower Z-levels, precluding their detonations being picked up by sensors. Basically, you get to test with one bomb if its on an upper level. This issue is also notable on the Gladius.

**Comment by jzburda** [comment]
> 
> 
> The toxins catapult was moved down a deck at the request of someone else to mitigate any damage that might occur on the top deck, which would breach lower decks and spray debris over them
> 
> I'll take the advice, I did forget to re-add a few items in various places



> 
> 
>     * Add leftover bodies to morgue
> 
>     * Engi warehouse sprites do not align with walls
> 
>     * Add more firelocks in very large rooms to reduce airloss
> 
>     * Remove plasma filter on constricted plasma
> 
>     * Engineering lost its fireaxe, bring it back
> 
>     * Add 3 oxy tanks to toxins since an RPD and easy distro access was not enough
> 
>     * Replace toxins ventpump with a scrubber
>     * Add some more random garbage to maintenance and random templates, mostly empty crates

Alright, I apologize if I was being snarky-- but why you gotta do me like that

**Comment by AU518987077** [comment]
atmospherics maintenance APC needs a knot, deck 2 starboard side, aft wall of chapel backroom, starboard of mess hall's aft maintenance door on its starboard wall.

**Comment by Vasily2013** [comment]
Debug tests

image

image

You seem to have forgotten intercomms in engineering

Obviously not all areas need RC's, but just double check that.

File: code/_DEFINES/NSV13_defines.dm [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #define MASS_TINY 1 //1 Player - Fighters
 #define MASS_SMALL 2 //2-5 Players - FoB/Mining Ship
 #define MASS_MEDIUM 3 //10-20 Players - Small Captial Ships
+#define MASS_MEDIUM_LARGE 3.5 //10-20 Players - Small Captial Ships
```
Comment (Bokkiewokkie) [inline code/_DEFINES/NSV13_defines.dm:4]: Capital 💢

**Comment by Kmc2000** [comment]
I’ll testmerge for this weekend

**Comment by Vasily2013** [comment]
I'm gonna do an integrity sweep on everything here in a little bit.

**Comment by Vasily2013** [comment]
- TEG power net doesn't start connected, and obvious connection point isn't connected to the SMES from what I could tell.
- plasma can in the RBMK room starts empty.
- Just make sure you clean up the science lobby pipes. 


This is space.
image

**Review by Kmc2000 — APPROVED** [review]

**Pull request merged**